app/src/main/java/org/simple/clinic/activity/TheActivity.kt

```diff
@@ -61,10 +62,10 @@ class TheActivity : AppCompatActivity() {
     }
 
     lifecycle
-        .startWith(TheActivityLifecycle.Started("TheActivity"))
+        .startWith(Started("TheActivity"))
         .compose(controller)
         .observeOn(mainThread())
-        .takeUntil(lifecycle.ofType<TheActivityLifecycle.Destroyed>())
+        .takeUntil(lifecycle.ofType<Destroyed>())
```

Comment:
Is there a reason why the `Destroyed` event isn't taking in an activity name?

Reply:
Oops... NVM.

Reply:
This is a type parameter and not an instance construction.